Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx

```diff
@@ -50,6 +50,7 @@ void ITSMFTDeadMapBuilder::init(InitContext& ic)
   mTFLength = ic.options().get<int>("tf-length");
   mDoLocalOutput = ic.options().get<bool>("local-output");
   mObjectName = ic.options().get<std::string>("outfile");
+  mCCDBUrl = ic.options().get<std::string>("ccdb-url");
```

Comment:
Note that the option may return `none` which should be treated as an empty string, better add
```
if (mCCDBUrl == "none") {
  mCCCDBUrl = "";
}
```